### PR TITLE
Enable learning rate decay to be disabled

### PR DESCRIPTION
**Pull request by ervteng**
Normally, learning rate is decayed until `max_steps` is reached. This is unnecessary (or even undesirable) for SAC as the Q functions will naturally converge over time, and we don't see the same instabilities in training. 

Removing this decay in SAC improves performance for WallJump and Crawler environments and doesn't regress performance for any other environments

In the interest of keeping the option list slim, learning_rate_schedule is an optional param (docs included).

### CODE REVIEW

File: ml-agents/mlagents/trainers/trainer.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 from mlagents.envs import UnityException, AllBrainInfo, ActionInfoOutputs, BrainInfo
 from mlagents.envs.timers import set_gauge
-from mlagents.trainers import TrainerMetrics
+from mlagents.trainers.trainer_metrics import TrainerMetrics
```
Comment (ervteng) [inline ml-agents/mlagents/trainers/trainer.py]: This fixes some import errors

File: docs/Training-SAC.md [hunk 2]
```diff
@@ -137,6 +137,18 @@ does not consistently increase.
 
 Typical Range: `1e-5` - `1e-3`
 
+### (Optional) Learning Rate Schedule
+
+`learning_rate_schedule` corresponds to how the learning rate is changed over time.
+For SAC, we typically hold learning rate constant so that the agent can continue to
```
Comment (awjuliani) [inline docs/Training-SAC.md]: Maybe instead of "we typically" we can say "we recommend."

File: ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py [hunk 3]
```diff
@@ -50,6 +50,11 @@ def create_model(
                             PPOModel(
                                 brain=brain,
                                 lr=float(trainer_params["learning_rate"]),
+                                lr_schedule=LearningRateSchedule(
+                                    trainer_params.get(
+                                        "learning_rate_schedule", "linear"
```
Comment (chriselion) [inline ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py]: This should LearningRateSchedule.LINEAR, right? (same for the other places that it's used)

Comment (ervteng) [reply ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py]: I'm using the same convention that the `vis_encode_type` uses - maybe we can change both?

Comment (chriselion) [reply ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py]: Oh my bad, it's creating a new enum instance there. All good!
